src/cli/helpers.ts

```diff
@@ -34,13 +36,15 @@ interface IParseGherkinMessageStreamRequest {
  * @param eventDataCollector
  * @param gherkinMessageStream
  * @param order
+ * @param unexpandedFeaturePaths
  * @param pickleFilter
  */
 export async function parseGherkinMessageStream({
```

Comment:
This function is deprecated and will be removed in a future major release, however in the meantime it'd be better to avoid making a breaking change to it. Could we make the new field optional?